views/widget.subscribe.display.php

```diff
@@ -306,7 +309,7 @@
 										echo 'required=required ';
 									}
 
-									echo 'type="checkbox" name="custom_' . esc_attr( $name_stripped ) . '[]" value="' . esc_attr( $vals[0] ) . '"> ' . esc_html( isset( $vals[1] ) ? $vals[1] : '' ) . '';
+									echo 'type="checkbox" name="custom_' . esc_attr( $name_stripped ) . $html_array. '" value="' . esc_attr( $vals[1] ) . '"> ' . esc_html( isset( $vals[0] ) ? $vals[0] : '' ) . '';
```

Comment:
WordPress.com VIP team here 👋 . We do embrace late escaping best practices whenever possible. Could we, perhaps, inline the value of `$html_array` here? Eg:

```
echo 'type="checkbox" name="custom_' . esc_attr( $name_stripped ) . ( count( $items ) > 1 ) ? '[]' : '' . '
```